mycobot_arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - mycobot_arm_joint_0
    - mycobot_arm_joint_1
    - mycobot_arm_joint_2
    - mycobot_arm_joint_3
    - mycobot_arm_joint_4
    - mycobot_arm_joint_5
  gains:
    mycobot_arm_joint_0:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    mycobot_arm_joint_1:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    mycobot_arm_joint_2:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    mycobot_arm_joint_3:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    mycobot_arm_joint_4:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    mycobot_arm_joint_5:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
    - joint_6
  gains:
    joint_1:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_2:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_3:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_4:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_5:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_6:
      p: 100
      d: 1
      i: 1
      i_clamp: 1